### ISSUE TITLE
Remove redundant alternate identifier form in register-dataset.cgi template

### ISSUE THREAD

**Issue by mbjones** [NCEAS/metacat#1184]
---

Author Name: **Chris Jones** (Chris Jones)
Original Redmine Issue: 7200, https://projects.ecoinformatics.org/ecoinfo/issues/7200
Original Date: 2017-06-19
Original Assignee: Chris Jones

---

We somehow have duplicated the @alternateIdentifier@ section of the @entryForm.tmpl@ template, which is causing display issues when viewing metadata documents.  After looking at the commit history, it looks like a merge issue.  Remove the duplicate form.

**Comment by mbjones** [comment]
---

Original Redmine Comment
Author Name: **Chris Jones** (Chris Jones)
Original Date: 2017-07-22T23:02:23Z

---

I removed the duplicated code in @metacatui/entryForm.tmpl@. Just needs to be merged into the 2.8 branch.

**Comment by mbjones** [comment]
---

Original Redmine Comment
Author Name: **Bryce Mecum** (Bryce Mecum)
Original Date: 2017-12-06T21:23:07Z

---

Found this issue while attempting to find out if the bug involving alternate identifiers auto-duplicating was fixed yet or not. This behavior appears to not have been fixed on KNB and I've also tested dev.nceas and see the issue there too. I'm not sure if the auto-duplicating of alternate identifiers is caused by the issue described in this ticket but it seems likely to be related.
